CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # CHANGELOG
+### [v4.18 _(Mar 8, 2022)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.18)
```

Comment:
Please also update the date too.

Reply:
NIce catch fixed -> https://github.com/omise/omise-woocommerce/pull/254/commits/7851ff5c67e9a52eca10ba088ff00cf00b43ce69 🙏